vyper/builtin_functions/functions.py

```diff
@@ -2007,6 +2039,10 @@ def build_LLL(self, expr, context):
     "bitwise_not": BitwiseNot(),
     "uint256_addmod": AddMod(),
     "uint256_mulmod": MulMod(),
+    "unsafe_add": UnsafeAdd(),
+    "unsafe_sub": UnsafeSub(),
+    "unsafe_mul": UnsafeMul(),
+    "unsafe_div": UnsafeDiv(),
```

Comment:
Wait, where are these classes being defined?

Reply:
Not defined yet, still WIP 😅

Reply:
Ah alright lol